File: 프로그래머스/unrated/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.cs
using System;
using System.Text;

public class Solution {
    public string solution(string my_string, string overwrite_string, int s) {
        string answer = "";
        StringBuilder str = new StringBuilder();
        str.Append(my_string.Substring(0, s));
        str.Append(overwrite_string);
        Console.Write(str.Length);
        Console.Write(my_string.Length);
        if(str.Length != my_string.Length)
        {
            str.Append(my_string.Substring(str.Length, my_string.Length - str.Length));
        }
        //Console.Write(my_string.Substring(str.Length, my_string.Length));
        
        answer = str.ToString();
        return answer;
    }
}

File: 프로그래머스/lv0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, string overwrite_string, int s) {
    string answer = "";
    answer.append(my_string.substr(0, s));
    answer.append(overwrite_string);
    answer.append(my_string.substr(answer.size(), my_string.size() - answer.size()));
    return answer;
}